### PR TITLE
docs: review of documentation

### PR DESCRIPTION
**Pull request by EazyAl**
I hope it works this time

### CODE REVIEW

File: docs/source/additional/faq.rst [hunk 1]
```diff
@@ -5,27 +5,26 @@ FAQ
...
+Substra is an open source federated learning (FL) software that enables machine learning on distributed datasets. It provides a flexible Python interface and a web app to perform federated machine learning at scale.
 
+Substra is the most proven software for federated learning in healthcare and has already been deployed in real production environments by hospitals and biotech companies (see the `MELLODDY <https://www.melloddy.eu/>`_ project for instance). Substra can also be used on a single machine on a virtually splitted dataset to perform FL simulations and debug code before launching experiments on a real network.
 
 Who owns Substra?
 ^^^^^^^^^^^^^^^^^
-Substra was originally developed by `Owkin <https://owkin.com/>`_ and is now hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_. Today Owkin is the main contributor to Substra.
+Substra is open source tool operated under an Apache 2.0 License. Substra is hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_. Substra was initially developed by engineers at `Owkin <https://owkin.com/>`_, a BioTech company that continues to play a big role in its development.
```
Comment (RomainGoussault) [inline docs/source/additional/faq.rst:14]:
```suggestion
Substra is open source software operated under an Apache 2.0 License. Substra is hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_. Substra was initially developed by engineers at `Owkin <https://owkin.com/>`_, a BioTech company that continues to play a big role in its development.
```

Comment (RomainGoussault) [reply docs/source/additional/faq.rst:14]: We refer Substra as "software". In the past we used to say "framework" so you might encounter this term sometimes.

File: docs/source/additional/faq.rst [hunk 2]
```diff
@@ -5,27 +5,26 @@ FAQ
...
 
 What kind of machine learning model can I use with Substra?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Substra is fully compatible with machine learning models written in Python from any library (PyTorch, Tensorflow, Sklearn, etc). However, a specific interface has been developed to use PyTorch in Substra, which makes writing PyTorch code simpler than using other frameworks.
+Substra is fully compatible with machine learning models written in any Python library (PyTorch, Tensorflow, Sklearn, etc). However, a specific interface has been developed to use PyTorch in Substra, which makes writing PyTorch code simpler than using other frameworks.
 
 Is Substra limited to medical and biotech applications?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Substra has been designed to work especially well on healthcare use cases; however, as Substra can work on any kind of data with any Python libraries, Substra can be used for any computation on distributed data.
+Although substra has been designed to work especially well in healthcare settings, it can work on any kind of data with any Python library to perform computation or analysis using distributed data.
```
Comment (RomainGoussault) [inline docs/source/additional/faq.rst:26]:
```suggestion
Although Substra has been designed to work especially well in healthcare settings, it can work on any kind of data with any Python library to perform computation or analysis using distributed data. 
```

File: docs/source/substrafl_doc/substrafl_overview.rst [hunk 3]
```diff
@@ -60,22 +60,23 @@ Evaluation Strategy
...
-A metric is a function that computes a performance by comparing labels (ground truth) and model's predictions.
+A metric is a function that computes performance by comparing the model's predictions against labelled data.
 One or several metrics can be added for an :ref:`concept_evaluation_strategy`.
 
 Index Generator
 ^^^^^^^^^^^^^^^
 
-The notion of epochs does not fully apply to the FL setting. Usually we don't want to train on a full epoch on each organization at every round, but on a reduced quantity of data to prevent models from different organizations from diverging too much.
-In a federated setting, at each round, in each organization, the model is trained for ``num_updates`` batches, with each batch containing ``batch_size`` data points.
+The notion of epochs does not fully apply to the FL setting. Training on a full epoch on each organization at every round can often result in divergence, which means a model may miss out on the potential advantages of having different datasets. To best utilize these datasets, we propose training your models on a subset of data within each dataset. This is further explained below (See diagram).
```
Comment (RomainGoussault) [inline docs/source/substrafl_doc/substrafl_overview.rst:75]: I would keep the previous formulation as:
- "can often result in divergence" seems weird. It could imply that your local loss does not converge but diverge.
- "we propose training" could imply that we came up with this idea, but actually everybody does that.

I would keep your last sentence though: "This is further explained below (See diagram)."

File: docs/source/substrafl_doc/substrafl_overview.rst [hunk 4]
```diff
@@ -90,22 +91,14 @@ Node
 ^^^^
 There are three types of node:
 
+.. warning:: The term 'node' is used interchangeably with 'organization' in Substra. Both terms represent different points in a federated data network.
```
Comment (RomainGoussault) [inline docs/source/substrafl_doc/substrafl_overview.rst:94]: I would remove this warning.
In the documentation and in the software there is not confusion between node and organization. However there is often confusion when we speak.

File: docs/source/substrafl_doc/substrafl_overview.rst [hunk 5]
```diff
@@ -119,6 +112,21 @@ Each round represents one iteration of the training loop in the federated settin
 
 Now that you have a good overview of SubstraFL, have a look at the :ref:`MNIST example <substrafl_doc/examples/get_started/plot_substrafl_torch_fedavg:Using Torch FedAvg on MNIST dataset>`.
```
Comment (RomainGoussault) [inline docs/source/substrafl_doc/substrafl_overview.rst:113]: To be put after the FL strategies paragraph

**Review by RomainGoussault — CHANGES_REQUESTED** [review]
Thanks for the PR. You did catch a lot of stuff!

Only minor comments.

**Comment by RomainGoussault** [comment]
Also could you please add a meaningful to this PR?

File: docs/source/additional/faq.rst (hunk 1 above)
Comment (ThibaultFy) [inline docs/source/additional/faq.rst:14]:
```suggestion
Substra is an open source tool operated under an Apache 2.0 License. Substra is hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_. Substra was initially developed by engineers at `Owkin <https://owkin.com/>`_, a BioTech company that continues to play a big role in its development.
```

File: docs/source/documentation/concepts.rst [hunk 6]
```diff
@@ -43,19 +43,20 @@ Compute plan and tasks
...
+^^^^^^^^^^^^^^^^
+
 A set of tasks.
 Gathering tasks into a single compute plan will lead to a more optimized compute.
 
 Note that you can register a task alone, i.e. not put the task in a compute plan, but Substra will still create a compute plan for you for this specific task.
 
-Task
-""""
+**Task**
```
Comment (ThibaultFy) [inline docs/source/documentation/concepts.rst:54]: Why did you change the title format to bold ?

File: docs/source/documentation/concepts.rst [hunk 7]
```diff
@@ -43,19 +43,20 @@ Compute plan and tasks
...
+
 A task correspond to a computation task. The task can use any inputs (data, algorithms or output from other tasks). The algorithm is expected to write the outputs in files, on paths given as outputs dictionary.
 
-Transient task outputs
-""""""""""""""""""""""
+**Transient task outputs**
```
Comment (ThibaultFy) [inline docs/source/documentation/concepts.rst:58]: Same here

**Review by ThibaultFy — COMMENTED** [review]
Thanks a lot for all the modifications !! :)

File: docs/source/documentation/concepts.rst (hunk 6 above)
Comment (EazyAl) [reply docs/source/documentation/concepts.rst:54]: It is hard to see that these are titles/headings because the text is almost the same size. Wasn't clear to me as I went through the documentation @ThibaultFy 
https://share.getcloudapp.com/geuOoe7P

Comment (ThibaultFy) [reply docs/source/documentation/concepts.rst:54]: Ok I see, there is a problem indeed, I'll make some suggestion to keep them a rst title

File: substrafl_examples/get_started/plot_substrafl_torch_fedavg.py [hunk 8]
```diff
@@ -3,16 +3,12 @@
...
 
-This example illustrates the basic usage of SubstraFL, and proposes Federated Learning model training using the Federated Average strategy
+This example illustrates the basic usage of SubstraFL and proposes Federated Learning using the Federated Averaging strategy
 on the `MNIST Dataset of handwritten digits <http://yann.lecun.com/exdb/mnist/>`__ using PyTorch.
-In this example, we work on 28x28 pixel sized grayscale images. The problem considered is a classification problem
-aiming to recognize the number written on each image..
+In this example, we work on 28x28 pixel sized grayscale images. This is a classification problem
+aiming to recognize the number written on each image.
 
-SubstraFL can be used with any machine learning framework (PyTorch, Tensorflow, Scikit-Learn, etc). However a specific
```
Comment (ThibaultFy) [inline substrafl_examples/get_started/plot_substrafl_torch_fedavg.py:11]: I think we should keep this in the example, even if it's already said elsewhere in the doc. The example will be the entry point for a lot of users and it is and important point to have in mind IMO.

**Review by ThibaultFy — COMMENTED** [review]
One minor comment and it'll be perfect for me :)
Also, as a detail, we don't use upper case in PRs name (you can rename it `docs: review of documentation`, it'll be great)

**Review by RomainGoussault — APPROVED** [review]
Looks good to me, thanks for the PR!

**Review by ThibaultFy — APPROVED** [review]
Great PR, thanks !! :)
